client/src/modules/prices/prices.service.js

```diff
@@ -78,5 +78,13 @@ function PriceListService(Api) {
     return service.$http.get(url, params)
       .then(service.util.unwrapHttpResponse);
   }
+
+  function downloadTemplate() {
+    const url = service.url.concat('download/template');
+    return service.$http.get(url)
+      .then(response => {
+        return service.util.download(response, 'Iventory item Template', 'csv');
```

Comment:
This should be spelled 'Inventory Item Template'.

Can you use `$translate` to provide a custom filename for the user?